Add WTD Portland 2025 schedule (#2334)
writethedocs/www: docs/conf/portland/2025/news/announcing-schedule.rst

:template: {{year}}/generic.html

> .. post:: Mar 13, 2025

Reviewer: ```suggestion
.. post:: Mar 18, 2025
```

   :tags: {{year}}, {{shortcode}}-{{year}}, schedule

Announcing Our Conference Schedule
========================

The {{city}} conference is just over six weeks away, and we're busy getting ready behind the scenes.

You've already seen our list of :doc:`awesome speakers </conf/portland/{{year}}/speakers/>`; now we're announcing our full conference schedule so you know when they will be presenting!

Speaker Schedule
--------------------------

The website is updated with the :doc:`schedule </conf/portland/{{year}}/schedule/>` for the full conference, including main stage speaker talks, so you can start planning your time (and getting excited!) for the conference. 

We'll continue to have a moderated Q&A session after each speaker. You'll have a chance to ask our speakers questions after the talk via our wonderful set of moderators. Virtual attendees can ask questions using the same QR code as folks attending locally.

> Now is a great time to buy your `tickets <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_.

Reviewer 2: Add the green button "Buy Your Tickets"

* **Sunday Welcome Reception:** Join us for our evening conference reception. This is a great chance to meet other attendees, get your badge (if you don't have it yet), and familiarize yourself with the venue. Drinks and snacks provided by us.
* **Monday Conference Party**. On Monday night we'll be hosting a social event off-site at `Jupiter NEXT <https://www.jupiterhotel.com/>`_, 900 E Burnside St, 2nd floor. This is a great chance to get to know more folks in the community and have a drink or two on us. Non-alcoholic options and snacks also provided. Event is 7-9 PM.

Docs in Discussion: Speaker Series
----------------------------------

**NEW this year:** We're hosting three pre-conference virtual discussions with a few of our speakers. Join us on March 24 with Manny Silva and Alex Garnett and April 1 with Jodie Landes. All dialogues are at 12 PM PST. `View more here <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/news/announcing-speaker-series/>`_ and `get your free ticket today <https://ti.to/writethedocs/write-the-docs-portland-2025/with/free-livestream>`_.

T-shirts Are Now Available
--------------------------

> The `Write the Docs {{city}} {{year}} Pop-Up Shop <https://shirt.writethedocs.org/>`_ is open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want. As in previous years, we're doing the pop-up shop option so people can order the exact shirt they want, if they want one.

Reviewer: ```suggestion
The `Write the Docs {{city}} {{year}} T-shirt Shop <https://shirt.writethedocs.org/>`_ is open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want. As in previous years, we're doing a mail order shop so people can order the exact shirt they want, if they want one.
```

The campaign will run in batches until May 31st. Shipping is available from the USA or Europe. We recommend you order your shirt at least 2-3 weeks in advance if you want to sport your fancy new garb during the conference.

Website Resources
-----------------

View our `Attendee Guide <https://www.writethedocs.org/conf/{{city}}/{{year}}/attendee-guide/>`_ for a comprehensive overview of how to participate in the conference.

Thanks To Our Sponsors
----------------------

Thanks to our sponsors for supporting the conference this year. A number of them will be tabling on Monday and Tuesday. We hope you get a chance to talk with them while you're here.

.. datatemplate::
   :source: /_data/{{shortcode}}-{{year}}-config.yaml
   :template: {{year}}/sponsors-simplelist.rst

Head over to our `sponsorship prospectus <https://www.writethedocs.org/conf/portland/{{year}}/sponsors/prospectus/>`_ for details on how your company can get involved!

See You Soon!
-------------

Thanks so much for all your support and enthusiasm. We're so excited to be with you all soon!

> Write the Docs Team

Reviewer: I think we've been leaving this off this year, because it feels a bit odd to sign emails these days. 

```suggestion
```
